scripts/hail_batch/hgdp1kg_tobwgs_pca/hgdp_1kg_tob_wgs_pca.py

```diff
@@ -0,0 +1,62 @@
+"""Generates loadings, scores, and eigenvalues for the HGDP,1KG, and tob-wgs dataset"""
+
+import click
+import pandas as pd
+import hail as hl
+from hail.experimental import lgt_to_gt
+
+GNOMAD_HGDP_1KG_MT = (
+    'gs://gcp-public-data--gnomad/release/3.1/mt/genomes/'
+    'gnomad.genomes.v3.1.hgdp_1kg_subset_dense.mt'
+)
+
+TOB_WGS = 'gs://cpg-tob-wgs-main/joint_vcf/v1/raw/genomes.mt'
+
+GNOMAD_LIFTOVER_LOADINGS = 'gs://cpg-reference/gnomad/gnomad_loadings_90k_liftover.ht'
+
+
+@click.command()
+@click.option('--output', help='GCS output path', required=True)
+def query(output):  # pylint: disable=too-many-locals
+    """Query script entry point."""
+
+    hl.init(default_reference='GRCh38')
+
+    hgdp_1kg = hl.read_matrix_table(GNOMAD_HGDP_1KG_MT)
+    tob_wgs = hl.read_matrix_table(TOB_WGS).key_rows_by('locus', 'alleles')
+    loadings = hl.read_table(GNOMAD_LIFTOVER_LOADINGS).key_by('locus', 'alleles')
+
+    # filter to loci that are contained in both tables and the loadings
+    hgdp_1kg = hgdp_1kg.filter_rows(
+        hl.is_defined(loadings.index(hgdp_1kg['locus'], hgdp_1kg['alleles']))
+        & hl.is_defined(tob_wgs.index_rows(hgdp_1kg['locus'], hgdp_1kg['alleles']))
+    )
+    tob_wgs = tob_wgs.semi_join_rows(hgdp_1kg.rows())
+
+    # Entries and columns must be identical
+    tob_wgs = tob_wgs.select_entries(GT=lgt_to_gt(tob_wgs.LGT, tob_wgs.LA))
+    hgdp_1kg = hgdp_1kg.select_entries(hgdp_1kg.GT)
+    hgdp_1kg = hgdp_1kg.select_cols()
```

Comment:
Just checking: `tob-wgs` doesn't have any column fields?

Reply:
yup, the only column is 's', the sample field